### ISSUE TITLE
[Feature] Support arm64 based cpu for macOS

### ISSUE THREAD

**Issue by huikaihoo** [guessi/kubectl-grep#40]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
Cannot use in arm64 based cpu for macOS. Any plan on support it?
Thanks.

```sh
$ krew install grep
Updated the local copy of plugin index.
Installing plugin: grep
W0905 09:22:33.335262   47104 install.go:164] failed to install plugin "grep": plugin "grep" does not offer installation for this platform
failed to install some plugins: [grep]: plugin "grep" does not offer installation for this platform
```

**Describe alternatives you've considered**

**Additional context**

**Comment by guessi** [comment]
@huikaihoo thanks for highlight that issue, I will tried to build for arm based at next release.

**Comment by guessi** [comment]
@huikaihoo just released [v1.8.0](https://github.com/guessi/kubectl-grep/releases/tag/v1.8.0). Please let me know if it works for you, thanks!
